specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/stable/2020-05-15/examples/EnvironmentsCreate.json

```diff
@@ -0,0 +1,66 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "environmentName": "env1",
+    "parameters": {
+      "location": "West US",
+      "kind": "Gen1",
```

Comment:
@sandshadow I think this would require some code changes, like adding "Gen1" and "Gen2" to the EnvironmentKind enum.

Reply:
Yes, you're right. We also need to discuss with Blair for his switch to GA api-version in Ibiza.